contrib/ec2/deis.template.json

```diff
@@ -78,6 +78,13 @@
       "AllowedValues" : [ "PV", "HVM" ],
       "ConstraintDescription" : "must be either PV or HVM"
     },
+    "EC2EBSVolumeType" : {
+      "Description" : "EC2 EBS VolumeType",
+      "Type": "String",
+      "Default": "gp2",
+      "AllowedValues" : [ "gp2", "io1", "standard" ],
```

Comment:
Also note: if we allow io1, it is also important to modify the autoscaling group's launch configuration to enable provisioned IOPs, or you're not getting the dedicated gig-e iSCSI interface to the provisioned IOPs EBS volumes which results in pretty much the same performance on the EBS volume had you not enabled IOPs on the volume at all